test/test.awsclient.ts

```diff
@@ -401,12 +401,12 @@ describe('AwsClient', () => {
         // Simulate error during region retrieval.
         const scope = nock(metadataBaseUrl)
           .get('/latest/meta-data/placement/availability-zone')
-          .reply(500);
+          .reply(400);
```

Comment:
Why were the status codes here all changed?

Reply:
Technically 500s are retryable and tests would fail with an unexpected error. As the tests want a error on the first call, I changed it to 400.